### PR TITLE
Improve getting started guide and user guide docs syntax highlighting

### PR DESCRIPTION
**Pull request by a-robinson**
@thockin @bgrant0607

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @satnam6502

**Comment by bgrant0607** [comment]
Please run hack/run-gendocs.sh:
```
1.24s$ PATH=$GOPATH/bin:./third_party/etcd:$PATH ./hack/verify-gendocs.sh
docs/getting-started-guides/fedora/fedora_ansible_config.md
----
blank-lines-surround-preformatted:
contents were modified
docs/getting-started-guides/fedora/flannel_multi_node_cluster.md
----
blank-lines-surround-preformatted:
contents were modified
docs/getting-started-guides/mesos.md
----
blank-lines-surround-preformatted:
contents were modified
docs/getting-started-guides/scratch.md
----
table-of-contents:
contents were modified
docs/getting-started-guides/vagrant.md
----
blank-lines-surround-preformatted:
contents were modified
docs/user-guide/limitrange/README.md
----
blank-lines-surround-preformatted:
contents were modified
docs/user-guide/namespaces/README.md
----
blank-lines-surround-preformatted:
contents were modified
FAIL: changes needed but not made due to --verify
/home/travis/gopath/src/github.com/GoogleCloudPlatform/kubernetes/docs/ is out of date. Please run hack/run-gendocs.sh
```

File: docs/getting-started-guides/azure.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ For more complete applications, please look in the [examples directory](../../ex
 
 ## Tearing down the cluster
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/azure.md]: FWIW, we mostly use ```bash

File: docs/getting-started-guides/centos/centos_manual_config.md [hunk 2]
```diff
@@ -191,7 +191,7 @@ done
 
 * Check to make sure the cluster can see the node (on centos-master)
 
-```
+```console
 kubectl get nodes
```
Comment (bgrant0607) [inline docs/getting-started-guides/centos/centos_manual_config.md]: Commands in console need `$` prompt

File: docs/getting-started-guides/coreos/azure/README.md [hunk 3]
```diff
@@ -149,13 +149,13 @@ You will need to open another terminal window on your machine and go to the same
...
-```
+```sh
 export AZ_VM_SIZE=Large
 ```
 
 Now, run scale script with state file of the previous deployment and number of nodes to add:
 
-```
+```console
 ./scale-kubernetes-cluster.js ./output/kube_1c1496016083b4_deployment.yml 2
```
Comment (bgrant0607) [inline docs/getting-started-guides/coreos/azure/README.md]: Needs `$`

File: docs/getting-started-guides/docker-multinode/master.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ or it may be something else.
 
 Now run flanneld itself:
 
-```sh
+```shsh
```
Comment (bgrant0607) [inline docs/getting-started-guides/docker-multinode/master.md]: Typo

File: docs/getting-started-guides/centos/centos_manual_config.md [hunk 5]
```diff
@@ -127,7 +127,7 @@ systemctl stop iptables-services firewalld
 
 * Edit /etc/kubernetes/apiserver to appear as such:
 
-```       
+```       sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/centos/centos_manual_config.md]: extra spaces

File: docs/getting-started-guides/centos/centos_manual_config.md [hunk 6]
```diff
@@ -163,7 +163,7 @@ done
 
 * Edit /etc/kubernetes/kubelet to appear as such:
 
-```       
+```       sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/centos/centos_manual_config.md]: extra spaces

File: docs/getting-started-guides/docker-multinode/master.md [hunk 7]
```diff
@@ -90,13 +90,13 @@ To re-configure Docker to use flannel, we need to take docker down, run flannel
 
 Turning down Docker is system dependent, it may be:
 
-```sh
+```shsh
```
Comment (bgrant0607) [inline docs/getting-started-guides/docker-multinode/master.md]: typo

File: docs/getting-started-guides/docker-multinode/master.md [hunk 8]
```diff
@@ -90,13 +90,13 @@ To re-configure Docker to use flannel, we need to take docker down, run flannel
...
 sudo /etc/init.d/docker stop
 ```
 
 or
 
-```sh
+```shsh
```
Comment (bgrant0607) [inline docs/getting-started-guides/docker-multinode/master.md]: typo

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 9]
```diff
@@ -58,7 +58,7 @@ Ansible will take care of the rest of the configuration for you - configuring ne
 
 A Kubernetes cluster requires etcd, a master, and n nodes, so we will create a cluster with three hosts, for example:
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: console

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 10]
```diff
@@ -83,7 +83,7 @@ Get the IP addresses from the master and nodes.  Add those to the `inventory` fi
 
 We will set the kube_ip_addr to '10.254.0.[1-3]', for now.  The reason we do this is explained later...  It might work for you as a default.
 
-```
+```sh
 [masters]
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: console

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 11]
```diff
@@ -103,7 +103,7 @@ If you already are running on a machine which has passwordless ssh access to the
 
 edit: group_vars/all.yml
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: console

or no formatting

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 12]
```diff
@@ -147,7 +147,7 @@ The IP address pool used to assign addresses to pods for each node is the `kube_
 
 For this example, as shown earlier, we can do something like this...
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: console

File: docs/getting-started-guides/fedora/fedora_ansible_config.md (hunk 11 above)
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: Ooops. Yaml.

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 13]
```diff
@@ -54,7 +54,7 @@ The kubernetes package provides a few services: kube-apiserver, kube-scheduler,
 
 Hosts:
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: no formatting

Comment (bgrant0607) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: sorry, no syntax highlighting

File: docs/getting-started-guides/fedora/flannel_multi_node_cluster.md [hunk 14]
```diff
@@ -173,35 +175,35 @@ FLANNEL_IPMASQ=false
 
 * Issue the following commands on any 2 nodes:
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/flannel_multi_node_cluster.md]: console

File: docs/getting-started-guides/fedora/flannel_multi_node_cluster.md [hunk 15]
```diff
@@ -173,35 +175,35 @@ FLANNEL_IPMASQ=false
...
 #docker run -it fedora:latest bash
 bash-4.3# 
 ```
 
 * This will place you inside the container. Install iproute and iputils packages to install ip and ping utilities. Due to a [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1142311), it is required to modify capabilities of ping binary to work around "Operation not permitted" error.
 
-```
+```sh
```
Comment (bgrant0607) [inline docs/getting-started-guides/fedora/flannel_multi_node_cluster.md]: console

File: docs/getting-started-guides/vagrant.md [hunk 16]
```diff
@@ -117,7 +117,7 @@ The master node instantiates the Kubernetes master components as pods on the mac
 
 To view the service status and/or logs on the kubernetes-master:
 
-```sh
+```console
 vagrant ssh master
```
Comment (bgrant0607) [inline docs/getting-started-guides/vagrant.md]: Command prompt?

File: docs/getting-started-guides/vagrant.md [hunk 17]
```diff
@@ -134,7 +134,7 @@ vagrant ssh master
 
 To view the services on any of the nodes:
 
-```sh
+```console
 vagrant ssh minion-1
```
Comment (bgrant0607) [inline docs/getting-started-guides/vagrant.md]: prompt

File: docs/user-guide/persistent-volumes/README.md [hunk 18]
```diff
@@ -53,7 +53,7 @@ support local storage on the host at this time.  There is no guarantee your pod
 
   
 
-```
+```console
 
 // this will be nginx's webroot
```
Comment (bgrant0607) [inline docs/user-guide/persistent-volumes/README.md]: As long as you're here:

Eliminate blank line

Change `//` to `#`

File: docs/user-guide/persistent-volumes/README.md [hunk 19]
```diff
@@ -120,9 +120,9 @@ kubernetes        component=apiserver,provider=kubernetes   <none>
 You should be able to query your service endpoint and see what content nginx is serving.  A "forbidden" error might mean you 
 need to disable SELinux (setenforce 0).
 
-```
+```console
```
Comment (bgrant0607) [inline docs/user-guide/persistent-volumes/README.md]: More blank lines before and after

File: docs/user-guide/resourcequota/README.md [hunk 20]
```diff
@@ -156,7 +156,7 @@ nginx-t9cap   1/1       Running   0          49s
 
 And if we print out our quota usage in the namespace:
 
-```shell
+```console
 kubectl describe quota quota --namespace=quota-example
```
Comment (bgrant0607) [inline docs/user-guide/resourcequota/README.md]: `$`

File: docs/user-guide/update-demo/README.md [hunk 21]
```diff
@@ -67,7 +67,7 @@ This can sometimes spew to the output so you could also run it in a different te
 Kubernetes repository. Otherwise you will get "404 page not found" errors as the paths will not match. You can find more information about `kubectl proxy`
 [here](../../../docs/user-guide/kubectl/kubectl_proxy.md).
 
-```
+```console
 $ kubectl proxy --www=examples/update-demo/local/ &
 + kubectl proxy --www=examples/update-demo/local/
```
Comment (bgrant0607) [inline docs/user-guide/update-demo/README.md]: Why is this line here?

**Comment by bgrant0607** [comment]
Thanks. Finished one pass.

**Comment by bgrant0607** [comment]
Still fails hack/verify-gendocs.sh

**Comment by a-robinson** [comment]
Thanks for the very thorough set of eyes :) Updated

**Comment by bgrant0607** [comment]
Thanks much. LGTM
